docs/indexing-afriberta.md

```diff
@@ -0,0 +1,38 @@
+# Indexing the AfriBERTa Corpus
+
+This document contains instructions to index the [AfriBERTa corpus](https://aclanthology.org/2021.mrl-1.11/) hosted on 
+huggingface at https://huggingface.co/datasets/castorini/afriberta-corpus. The corpus contains train and evaluation sets
+in 10 African languages.
+
+# Languages
+- Afaanoromoo
+- Amharic
+- Gahuza
+- Hausa
+- Igbo
+- Pidgin
+- Somali
+- Swahili
+- Tigrinya
+- Yoruba
+
+# Downloading the dataset
+-  You can download the dataset from huggingface for both train and eval using the code below
+    ```bash
+    wget https://huggingface.co/datasets/castorini/afriberta-corpus/resolve/main/{$language}/train.zip
```

Comment:
Provide exactly the string to use for `$language`? Lower-case? two- or three-letter codes? etc.?

Reply:
Seems to be the string here? https://huggingface.co/datasets/castorini/afriberta-corpus/tree/main

Maybe mention?